includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -180,6 +180,58 @@ public function sendTemporaryPasswordEmail() {
 		}
 	}
 
+	/**
+	 * Api endpoint to send a password reset e-mail.
+	 * @requestParam user_id : \User::id
+	 * @requestParam token : user identification token
+	 */
+	public function sendPasswordResetLinkEmail() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)